src/UI/home/dashboard/homeContext.tsx

```diff
@@ -1,30 +1,100 @@
-import React from "react";
-import {Chart} from "./chart";
+import React, { useEffect, useState } from 'react'
```

Comment:
file should be named HomeContext. all files that have components in them should be named with PascalCase as well as component names inside of them. variables and functions names should use camelCase and variables that are global should use SCREAMING_SNAKE_CASE

Reply:
also interfaces and types should use PascalCase. personally i always name types and interfaces with 'I' in the front eg. `interface IHomeContextProps {...}`